src/renderer/dx/DxFontRenderData.cpp

```diff
@@ -46,28 +74,73 @@ DxFontRenderData::DxFontRenderData(::Microsoft::WRL::ComPtr<IDWriteFactory1> dwr
     return _lineMetrics;
 }
 
-[[nodiscard]] Microsoft::WRL::ComPtr<IDWriteTextFormat> DxFontRenderData::DefaultTextFormat() noexcept
+[[nodiscard]] Microsoft::WRL::ComPtr<IDWriteTextFormat> DxFontRenderData::DefaultTextFormat()

```

Comment:
I feel like I'm abusing the lazy-initializing pattern but it's really handy in this specific class and really helps with the structure.